DigitalLearningSolutions.Web/Controllers/TrackingSystem/Delegates/CourseDelegatesController.cs

```diff
@@ -89,5 +93,23 @@ public IActionResult AllCourseDelegates(int customisationId)
 
             return View(model);
         }
+
+        [ServiceFilter(typeof(VerifyAdminUserCanViewCourse))]
+        [Route("DownloadCurrent/{customisationId:int}")]
+        public IActionResult DownloadCurrent(int customisationId)
+        {
+            var centreId = User.GetCentreId();
+            var adminCategoryId = User.GetAdminCourseCategoryFilter();
+            var content = courseDelegatesDownloadFileService.GetCourseDelegateDownloadFileForCourse(
+                customisationId,
+                centreId,
+                adminCategoryId
+            );
+            return File(
+                content,
+                FileHelper.ExcelContentType,

```

Comment:
There is a library of media types provided in `.net core` albeit slightly more convoluted to obtain one.
Maybe the file helper could under the hood utilise that library meaning calls to things like `FileHelper.ExcelContentType`
would resolve to a method that did :
```
  if (new FileExtensionContentTypeProvider().TryGetContentType(fileName, out var contentType) {
    return contentType;
  }
```

Not necessary to implement but does avoid us storing our own versions of these `mime-types`.